src/main/kotlin/no/nav/syfo/motebehov/api/internad/v2/MotebehovVeilederADControllerV2.kt

```diff
@@ -25,16 +28,25 @@ class MotebehovVeilederADControllerV2 @Inject constructor(
     private val metric: Metric,
     private val historikkService: HistorikkService,
     private val motebehovService: MotebehovService,
+    private val pdlConsumer: PdlConsumer,
     private val veilederTilgangConsumer: VeilederTilgangConsumer
 ) {
     @GetMapping(value = ["/motebehov"], produces = [MediaType.APPLICATION_JSON_VALUE])
     fun hentMotebehovListe(
         @RequestParam(name = "fnr") sykmeldtFnr: @Pattern(regexp = "^[0-9]{11}$") String
-    ): List<Motebehov> {
+    ): List<MotebehovVeilederDTO> {
         metric.tellEndepunktKall("veileder_hent_motebehov")
         val fnr = Fodselsnummer(sykmeldtFnr)
```

Comment:
Vi forholder oss vanligvis til `fnr`. Hvis dette er variabel av type `Fodslsnummer`, bør den hete `fodselsnummer` slik den er i  `PdlConsumer.kt `:
```
fun person(fodselsnummer: Fodselsnummer): PdlHentPerson? {
        return person(fodselsnummer.value)
    }
```

Reply:
Fint det, men denne kodelinjen er ikke en del av denne PR'en, så om det er ønskelige med refaktoring av eksisterende kode så foreslår at man gjør det i egen PR 👍 
Her lager jeg bare et eget DTO-objekt for API'et til veileder og legger navn på den som opprettet Motebehov'et.

Reply:
Ordnet det nå.